scripts/install

```diff
@@ -13,6 +13,9 @@ source /usr/share/yunohost/helpers
 # MANAGE SCRIPT FAILURE
 #=================================================
 
+ynh_clean_setup () {
```

Comment:
Not needed as stated by @tituspijean here : https://github.com/YunoHost-Apps/dokuwiki_ynh/pull/92#issuecomment-1223069994